src/NBitcoin.Tests/NodeBuilderStratis.cs

```diff
@@ -109,8 +109,6 @@ public CoreNodeStratis CreateNode(bool start = false)
 			//}
 			var node = new CoreNodeStratis(child, this);
 			Nodes.Add(node);
-			if(start)
```

Comment:
is it never used?

Reply:
the code compiles, the code runs, all calls to here were with start = false ... it is not used because if it was, those uninitialized variables i removed would probably caused troubles, + this is some nbitcoin test only ... we do have our own tests for running nodes and + we will redesign node class ...